abci/example/dummyapp/main.go

```diff
@@ -0,0 +1,106 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"time"
+
+	dummy "github.com/lazyledger/lazyledger-core/abci/example/dummyapp/app"
+	cmd "github.com/lazyledger/lazyledger-core/cmd/tendermint/commands"
+	"github.com/lazyledger/lazyledger-core/cmd/tendermint/commands/debug"
+	cfg "github.com/lazyledger/lazyledger-core/config"
+	"github.com/lazyledger/lazyledger-core/ipfs"
+	"github.com/lazyledger/lazyledger-core/libs/cli"
+	"github.com/lazyledger/lazyledger-core/libs/log"
+	"github.com/lazyledger/lazyledger-core/node"
+	"github.com/lazyledger/lazyledger-core/p2p"
+	"github.com/lazyledger/lazyledger-core/privval"
+	"github.com/lazyledger/lazyledger-core/proxy"
+)
+
+var (
+	randTxs  uint32
+	txSize   uint32
+	msgSize  uint32
+	randMsgs uint32
+
+	sleepDuringPreprocess time.Duration
+
+	runNodeCmd = cmd.NewRunNodeCmd(DummyNode)
+)
+
+func init() {
+	runNodeCmd.Short = "Run dummyapp"
+
+	runNodeCmd.PersistentFlags().DurationVar(
+		&sleepDuringPreprocess,
+		"dummy.sleep", 250*time.Millisecond,
+		"duration to sleep during preprocess phase, e.g. 500ms, 1s")
+
+	runNodeCmd.PersistentFlags().Uint32Var(&randTxs, "dummy.txs", 10, "generate random transactions")
+	runNodeCmd.PersistentFlags().Uint32Var(&txSize, "dummy.tx-size", 50, "size of random transactions")
+
+	runNodeCmd.PersistentFlags().Uint32Var(&randMsgs, "dummy.msgs", 16, "generate random messages")
+	runNodeCmd.PersistentFlags().Uint32Var(&txSize, "dummy.msg-size", 128, "size of random messages")
```

Comment:
With these flags, we can experiment with different block sizes (e.g. for the light client).